en/document-summaries.html

```diff
@@ -6,117 +6,220 @@
-  To include the <em>document ID</em> in a custom summary
-  class, add a field for the id and include it in the summary class.
+  In addition to the schema fields,
+  one can also select <a href="reference/default-result-format.html#sddocname">sddocname</a>
+  and <a href="reference/default-result-format.html#documentid">documentid</a>:
 </p>
+<pre>
+$ vespa query "select <span class="pre-hilite">artist</span>, <span class="pre-hilite">album</span>, <span class="pre-hilite">documentid</span>, <span class="pre-hilite">sddocname</span> from music where album contains 'head'"
+</pre>
+<pre>{% highlight json %}
+{
+    "root": { },
+        "children": [
+            {
+                "id": "id:mynamespace:music::a-head-full-of-dreams",
+                "relevance": 0.16343879032006284,
+                "source": "music",
+                "fields": {
+                    "sddocname": "music",
+                    "documentid": "id:mynamespace:music::a-head-full-of-dreams",
+                    "artist": "Coldplay",
+                    "album": "A Head Full of Dreams"
+                }
+            }
+        ]
+    }
+}
+{% endhighlight %}</pre>
+<!-- ToDo: Add a detail on the id-field that in the latter case is the document-id -->
 <p>
-  Use <a href="reference/schema-reference.html#summary">dynamic</a>
-  to generate dynamic abstracts of fields, based on search keywords.
+  Use <code>*</code> to select <span style="text-decoration: underline">all</span> fields.
+  As no <a href="#summary-classes">summary class</a> is given,
+  the <em>default</em> summary class with all fields is used:
 </p>
+<pre>
+$ vespa query "select <span class="pre-hilite">*</span> from music where album contains 'head'"
+</pre>
+<pre>{% highlight json %}
+{
+    "root": { },
+        "children": [
+            {
+                "id": "id:mynamespace:music::a-head-full-of-dreams",
+                "relevance": 0.16343879032006284,
+                "source": "music",
+                "fields": {
+                    "sddocname": "music",
+                    "documentid": "id:mynamespace:music::a-head-full-of-dreams",
+                    "artist": "Coldplay",
+                    "album": "A Head Full of Dreams",
+                    "year": 2015,
+                    "category_scores": {
+                        "type": "tensor<float>(cat{})",
+                        "cells": {
+                            "pop": 1.0,
+                            "rock": 0.20000000298023224,
+                            "jazz": 0.0
+                        }
+                    }
+                }
+            }
+        ]
+    }
+}
+{% endhighlight %}</pre>
+<p>
+  Query performance depends on the fields returned, see the <a href="#performance">performance</a> section.
+</p>
+
 
 
 
-<h2 id="defining-summary-sets-in-the-schema">Defining summary sets in the schema</h2>
+<h2 id="dynamic-summaries">Dynamic summaries</h2>
```

Comment:
I think summary classes (#summary-classes) is more important than this section, and is a more natural continuation of the above YQL examples. Consider switching the order.

Reply:
Yes, that will make the flow better! I am not 100% sure, though, as the dynamic part is a functionally important piece, maybe more so than the classes, which is more of an optimization.

As I will rewrite the intro, I will consider and most likely follow your suggestion in next PR. 

Thanks for great feedback!